src/Apps/Artist/Components/ArtistHeader/ArtistHeader2.tsx

```diff
@@ -235,10 +228,7 @@ export const ArtistHeaderFragmentContainer = createFragmentContainer(
           credit
         }
         insights {
-          kind
```

Comment:
You still need `kind` — surprised this didn't type error?

Reply:
Its in `ArtistCareerHighlight_insight`

Reply:
That should absolutely type error — you cannot reach into fragments — lemme pull down and see what's going on

Reply:
WOW